browser/ui/BUILD.gn

```diff
@@ -240,6 +241,14 @@ source_set("ui") {
       "toolbar/brave_bookmark_sub_menu_model.cc",
       "toolbar/brave_bookmark_sub_menu_model.h",
       "toolbar/brave_recent_tabs_sub_menu_model.h",
+      "webui/brave_education/brave_education_ui.cc",
```

Comment:
new source files should not be added here. Please create a new BUILD.gn in webui/brave_education and use allow_circular_includes from

Reply:
Done.